rdr_service/dao/questionnaire_response_dao.py

```diff
@@ -922,6 +923,20 @@ def _update_participant_summary(
                         answer_value = code_dao.get(answer.valueCodeId).value
                         if answer_value.lower() == WEAR_YES_ANSWER_CODE:
                             self.consents_provided.append(ConsentType.WEAR)
+                        # participant_summary should contain most recently authored WEAR consent details
+                        if (participant_summary.consentForWearStudyAuthored is None
+                                or questionnaire_response.authored > participant_summary.consentForWearStudyAuthored):
+                            participant_summary.consentForWearStudyTime = questionnaire_response.created
+                            participant_summary.consentForWearStudyAuthored = authored
+                            if answer_value.lower() == WEAR_YES_ANSWER_CODE:
+                                participant_summary.consentForWearStudy = QuestionnaireStatus.SUBMITTED
+                            elif answer_value.lower() == WEAR_NO_ANSWER_CODE:
+                                participant_summary.consentForWearStudy = QuestionnaireStatus.SUBMITTED_NO_CONSENT
+                            else:
+                                logging.warning(
+                                    f'Unrecognized/no answer value for WEAR consent question: {answer_value} ',
+                                )
+                                participant_summary.consentForWearStudy = QuestionnaireStatus.UNSET
```

Comment:
In this case, should we set the consent status to SUBMITTED_INVALID instead?

Reply:
@kenny-skaggs INVALID can now imply a problem with consent PDFs (even if it's only used for EHR right now).  When that was use case was introduced, the PDR generators started using UNSET if we couldn't figure out how a participant responded to a consent question.

Reply:
I still think an INVALID would be good in this case. But this works too.

Reply:
I thought that our data dictionary/reference documentation now defined SUBMITTED_INVALID as "did not pass PDF validation", but maybe it can be a broader error indication.